org.osgi.test.junit5.cm/test.bndrun

```diff
@@ -51,4 +51,6 @@
 	org.osgi.service.cm;version='[1.6.0,1.6.1)',\
 	org.osgi.test.common;version='[0.10.0,0.10.1)',\
 	org.osgi.test.junit5.cm;version='[0.10.0,0.10.1)',\
-	org.osgi.test.junit5.cm-tests;version='[0.10.0,0.10.1)'
+	org.osgi.test.junit5.cm-tests;version='[0.10.0,0.10.1)',\
+	org.osgi.test.assertj.framework;version='[0.10.0,0.10.1)',\
```

Comment:
Please sort.

Reply:
maybe this would be a nice option for bnd

Reply:
Ask, and you shall receive:
```
-runstartlevel: \
    order=sortbynameversion,\
    begin=-1
```
Add the above to `test.bndrun` (before the -runbundles directive would be best, I think) and you should be good to go!

Reply:
I'm not sure why we haven't yet done this for the other bndrun files. Perhaps also a common `.bndrun` somewhere in cnf/includes might be nice so that we can reference this (and other common configuration) in all our bndrun files.